src/Types/IDEntityVin.php

```diff
@@ -85,9 +97,13 @@ public function isChecksumValidated(): bool
      */
     public function isValid(): bool
     {
-        /** @var VinCodeValidatorExtension $validator */
-        $validator = static::getContainer()->make(VinCodeValidatorExtension::class);
+        if (\is_string($this->value) && $this->value !== '') {
+            /** @var VinCodeValidatorExtension $validator */
+            $validator = static::getContainer()->make(VinCodeValidatorExtension::class);
+
+            return \is_string($this->value) && $validator->passes('', $this->value);
```

Comment:
Same. Redundant `\is_string($this->value)`

Reply:
Also fixed